js/src/lib/components/InferenceWidget/shared/types.ts

```diff
@@ -1,14 +1,16 @@
 import type { ModelData } from "../../../interfaces/Types";
 
 export interface WidgetProps {
-	apiToken?:          string;
-	apiUrl:             string;
-	callApiOnMount:     boolean;
-	model:              ModelData;
-	noTitle:            boolean;
-	shouldUpdateUrl:    boolean;
-	includeCredentials: boolean;
-	isLoggedIn?:        boolean;
+	accept?:               string;
+	apiToken?:             string;
+	apiUrl:                string;
+	callApiOnMount:        boolean;
+	isInferenceEndpoints?: boolean;
+	model:                 Pick<ModelData, 'id' | 'mask_token' | 'pipeline_tag' | 'widgetData'>;
```

Comment:
Allow passing only the attributes that are actually needed - required for hf-endpoints. Doesn't change the widget behavior.